src/Administration/Resources/app/administration/src/module/sw-users-permissions/components/sw-users-permissions-user-listing/sw-users-permissions-user-listing.html.twig

```diff
@@ -156,22 +156,6 @@
                 </p>
                 {% endblock %}
 
-                <!-- eslint-disable-next-line sw-deprecation-rules/no-twigjs-blocks -->
-                {% block sw_settings_user_list_delete_modal_input__confirm_password %}
```

Comment:
removing a block is a breaking change, so please keep the block (empty) and mark it as deprecated for 6.5.0, so that it will be removed with that version